homeassistant/helpers/update_coordinator.py

```diff
@@ -131,6 +137,10 @@ def _schedule_refresh(self) -> None:
                 self.logger.error("Error fetching %s data: %s", self.name, err)
                 self.last_update_success = False
 
+        except NotImplementedError as err:
+            self.last_update_success = False
```

Comment:
I think that we can even skip this, the person writing this code will never end up checking this if their whole class crashed ?

Reply:
Removed.